src/core/config/createCoreConfigs.js

```diff
@@ -38,6 +38,7 @@ export default () =>
     orgId: string()
       .unique()
       .required(),
+    personalizationStorageEnabled: boolean().default(true),
```

Comment:
Since it is a Beta feature, in my opinion this should be `disabled` by default, as we don't want to impact all the customers.

cc: @jfkhoury @jonsnyder

Reply:
decision was made to be enabled by default.